### ISSUE TITLE
Talkback does not work properly

### ISSUE THREAD

**Issue by ramazancilasin** [ceryle/SegmentedButton#16]
Hi, when I create a segmented button group, add some buttons programmatically (not in the xml) and turn on talkback from phone settings it does not work properly. Firstly, when on a button it says double tap to activate. Then, when all buttons are done with talkback and right swipe gesture is used to move to the next item, it does not continue with the next item, instead it goes back at the first button and only says the "Name of the button" + " button". Shouldn't it work like "Name of the button" + "button and double tap to activate?

**Comment by ceryle** [comment]
Hi,
I don't know the reason and also I haven't used TalkBack before. However, I will take a look at that issue too when I work on the new version.
